### PR TITLE
Bindings/get message id

### LINKED ISSUE #579: Add a function to the bindings to get the message id from a message
## Description

Function to get the message id from a message

## Motivation

One can receive messages with MQTT like 
```JSON
{"network_id":1454675179895816119,"parents":["6f67cff1ada4707f91dd5b282e011dc6584ba1ba9d207e15b2c49ca353a76b42","87b07566cc083ce866348dec078d76a2ed9aaf65ffb6aff58fea72ecac43bd84","94dd6b75d37ba45c542f7c7aef085a66ee44f0ce45d27aafcfcc5c227daaf0d1","bbb0fbb877e378a724e34cc32c334da94535e03acdee361fc8884de7f93cb352"],"payload":{"type":"Indexation","data":{"index":[72,79,82,78,69,84,32,83,112,97,109,109,101,114],"data":[73,79,84,65,32,45,32,65,32,110,101,119,32,100,97,119,110,10,67,111,117,110,116,58,32,49,54,48,57,57,49,53,10,84,105,109,101,115,116,97,109,112,58,32,50,48,50,49,45,48,53,45,50,53,84,49,48,58,50,48,58,51,49,43,48,50,58,48,48,10,84,105,112,115,101,108,101,99,116,105,111,110,58,32,52,55,194,181,115]}},"nonce":1356669}
```
but has no way to get the message id 

## Requirements

1. Accept a a string with the message object and return the message id
In Rust we can get it with `message.id().0`

## Are you planning to do it yourself in a pull request?

Yes.

### PR DESCRIPTION
**Pull request by Thoralf-M**
Python binding is missing

### CODE REVIEW

**Comment by bingyanglin** [comment]
for the Python binding, a message object will be returned as a dictionary, so we can use `message['message_id']` to get it.
An example follows:
```python
import iota_client
client = iota_client.Client()

message_id = "a3a5193f917ea82967faffbd35675d05553d676a078a57cf996d9c15e8774d2c"
message = client.get_message_data(message_id)
message_meta = client.get_message_metadata(message_id)

print("message_id in Message meta data:")
print(message_meta['message_id'])
print("message_id in Message data:")
print(message['message_id'])
```
The output will be
```b
message_id in Message meta data:
a3a5193f917ea82967faffbd35675d05553d676a078a57cf996d9c15e8774d2c
message_id in Message data:
a3a5193f917ea82967faffbd35675d05553d676a078a57cf996d9c15e8774d2c
```

**Comment by Thoralf-M** [comment]
@bingyanglin but not for messages from MQTT

File: bindings/python/native/src/client/high_level_api.rs [hunk 1]
```diff
@@ -163,6 +165,22 @@ impl Client {
         })?;
         Ok(children.iter().map(|child| hex::encode(child.as_ref())).collect())
     }
+    /// Get the message id from the payload string.
+    ///
+    /// Args:
+    ///     payload_str (str): The identifier of message.
+    ///
+    /// Returns:
+    ///     children ([str]): The returned list of children string.
```
Comment (Thoralf-M) [inline bindings/python/native/src/client/high_level_api.rs]: Should this be `message_id (str): The identifier of message.`?

Comment (Thoralf-M) [inline bindings/python/native/src/client/high_level_api.rs]:
```suggestion
    ///     message_id (str): The identifier of message.
```

Comment (bingyanglin) [reply bindings/python/native/src/client/high_level_api.rs]: Yes. Thanks for identifying this.

**Review by bingyanglin — APPROVED** [review]

**Pull request merged**